pages/common/jest.md

```diff
@@ -0,0 +1,23 @@
+# jest
+
+> Delightful JavaScript Testing.
```

Comment:
Please change this to something more objective, like - "A framework for testing javascript code" or something similar.

Reply:
This is, as far as I can tell, the project tagline, but I appreciate it is highly subjective. Will look into authoring a concise description. 👍

Reply:
We always try to explain a command in our own way. Otherwise, it would be just another man page. In all our pages, we try to keep the descriptions as objective as possible.